src/applications/disability-benefits/all-claims/containers/ITFWrapper.jsx

```diff
@@ -19,8 +19,12 @@ import {
 
 const fetchWaitingStates = [requestStates.notCalled, requestStates.pending];
 
-const noITFPages = ['/introduction', '/confirmation'];
 export class ITFWrapper extends React.Component {
+  static defaultProps = {
+    noITFPages: [/\/introduction/, /\/confirmation/],
+    /* noITFPages: ['/introduction', '/confirmation'], */
```

Comment:
Might not need the strings anymore

Reply:
D'oh. Testing leftovers. Not too tasty cold.

Good call.